### ISSUE TITLE
Color Scheme Generator for Tailwind

### ISSUE THREAD

**Issue by adevade** [tailwindcss/discuss#128]
Hey Tailwinders! 

Just wanted to share a tool I built (mainly for my own use) to quickly generate some UI colors. 

Enter a "brand color" (or randomize one), and you'll get a bunch of matching colors and greys. Then you can just copy the output to your tailwind.js! 

Both the colors (Call-To-Action, Info, Warning, Success, Danger) and the greys match the hue of your main brand color. 

Check it out: https://adevade.github.io/color-scheme-generator 

If you have any suggestions or ideas, I would love contributions! 
Happy coding <3

**Comment by ahuggins** [comment]
@adevade This is great! I was looking for something like this. I also found 0to255.com, which can help in picking your primary color. The two together is very neat. Anyway, posting in case anyone else stumbles on this post.

**Comment by robjstanley** [comment]
Thanks for this 🔥

**Comment by adevade** [comment]
@kiru Thank you! Yes, of course! I'll push an update later today!

**Comment by kiru** [comment]
You are right, it works! I did not realize that grouping in colors was not necessary. Thank you.

**Comment by oliverpool** [comment]
Hi,

I just found about this one: https://javisperez.github.io/tailwindcolorshades/

Which looks perfect to adjust some colors (for example if my brand is a blue that is not in tailwind palette, I can simply adjust all the blues of the tailwind config!)
